content/cloud/noops-sql-db.md

```diff
@@ -69,34 +67,41 @@ name = "c3"
 sqlite_databases = ["default"]
 ```
 
-The database will be created when the Spin application is deployed. The data persists Spin application invocations and updates. 
+The above configuration will last the lifespan of the application.
 
-```bash
-$ spin cloud deploy
-Deploying app ..
-Creating database inspirational-pig (default) for app "quickstart"
-```
+### Preparing Fermyon Cloud Schema
 
-You can check to see your database by using the `spin cloud sqlite` command:
+Bootstrapping your Cloud SQLite database schema is done via the `spin cloud sqlite execute` subcommand. Your NoOps SQLite application on localhost will have already assigned a name for your database. All we need to do is obtain the name by typing the following command (in your local NoOps SQLite application's directory):
 
->> Fermyon Cloud will automatically generate a random name for your database.
+<!-- @selectiveCpy -->
 
 ```bash
 $ spin cloud sqlite list
-inspirational-pig (default) 
+Databases (1)
+inspirational-pig (default)
+```
+
+Once we have this name (in this above example, we can see our database called `inspirational-pig`), we use that `inspirational-pig` as part of our `spin cloud sqlite execute` subcommand to create our schema for our Fermyon Cloud application:
+
+```bash
+$ spin cloud sqlite execute inspirational-pig "CREATE TABLE IF NOT EXISTS todos (id INTEGER PRIMARY KEY AUTOINCREMENT,description TEXT NOT NULL,due_date DATE,starred BOOLEAN DEFAULT 0,is_completed BOOLEAN DEFAULT 0)"
```

Comment:
Does `spin cloud sqlite execute` support `@file` syntax?  (Like `spin up --sqlite` does.)  If it does, it might be easier to show that than squeezing this long statement onto one line.

Reply:
I checked, unfortunately, it does not. Would be great to have though.

Reply:
Sigh, yeah... the `@file` syntax is there in the cloud-plugin code (https://github.com/fermyon/cloud-plugin/blob/0ce1b4e234471a46e9a9b6688c1b24256259a597/src/commands/sqlite.rs#L101), but looks like it went in a few hours after the 0.1.2 release.